esg_node.py

```diff
@@ -177,10 +176,6 @@ def system_component_installation(esg_dist_url, node_type_list):
         print "\n*******************************"
         print "Installing Data Node Components"
         print "******************************* \n"
-        try:
-            pip._internal.main(['install', "esgprep=={}".format(config["esgprep_version"])])
```

Comment:
Is esgprep getting installed elsewhere now?

Reply:
Responding to both comments in one. Yes it is in the publisher, this is from the esgcet `setup.py`
```python
setup(
    name = 'esgcet',
    version = VERSION,
    description = 'ESGCET publication package',
    author = 'PCMDI Software Team',
    author_email = 'webmaster@pcmdi.llnl.gov',
    url = 'http://esgf.llnl.gov',
    install_requires = [
        "psycopg2>=2.0",
        "SQLAlchemy>=0.7.8,!=0.9.5",
        "lxml>=2.0",
        "sqlalchemy_migrate>=0.6",
        "requests>=1.2.0",
        "esgprep>=2.8.1",
        "MyProxyClient>=2.0.1",
        "esgf-pyclient>=0.1.8",
        "esgfpid>=0.7.11",
        "cdf2cim>=0.1.9.0",
    ],
    setup_requires = [
        "psycopg2>=2.0",
        "SQLAlchemy>=0.7.8,!=0.9.5",
        "lxml>=2.0",
        "sqlalchemy_migrate>=0.6",
        "requests>=1.2.0",
],
...
```

Reply:
It actually looks like we can get rid of `lxml` as well. @William-Hill

Reply:
Actually `lxml` is being used by the installer, so better have it. But `myproxyclient` is not used.

Reply:
Sounds good.  I'll approve and merge